homeassistant/components/fully_kiosk/config_flow.py

```diff
@@ -41,10 +41,15 @@ async def async_step_user(
             try:
                 async with timeout(15):
                     device_info = await fully.getDeviceInfo()
-            except (ClientConnectorError, FullyKioskError, asyncio.TimeoutError):
+            except (
+                ClientConnectorError,
+                FullyKioskError,
+                asyncio.TimeoutError,
+            ) as error:
+                LOGGER.debug(error, exc_info=True)
                 errors["base"] = "cannot_connect"
-            except Exception:  # pylint: disable=broad-except
-                LOGGER.exception("Unexpected exception")
+            except Exception as error:  # pylint: disable=broad-except
+                LOGGER.debug(error, exc_info=True)
```

Comment:
This is an unknown exception. We still want that to be logged at error level with the stack trace.
```suggestion
                LOGGER.exception("Unexpected exception: %s", error)
```